src/tipowof10.c

```diff
@@ -175,6 +222,9 @@ const __int128 tifrexpof10 [] =
   (__int128 ) 6103515625000000ll, /* 10**20 */
   (__int128 ) 7629394531250000ll, /* 10**21 */
   (__int128 ) 4768371582031250ll, /* 10**22 */
+
+#ifdef __clang__
+#else
```

Comment:
`#ifndef __clang__` instead?
also, why? add a comment.

Reply:
// Clang does support constant folding for __int128
// Simply not posible to defind a const larger than __UINT64_MAX__
// For vector initialized can use two long long elements.